public/mcaptcha-glue.js

```diff
@@ -0,0 +1 @@
rror("Class extends value "+String(t)+" is not a constructor or null");function i(){this.constructor=e}n(e,t),e.prototype=null===t?Object.create(t):(i.prototype=t.prototype,new i)});Object.defineProperty(t,"__esModule",{value:!0}),t.ConfigurationError=void 0;var o=function(e){function t(){var t=null!==e&&e.apply(this,arguments)||this;return t.message="Provide either widget link or site key to display mCaptcha widget",t}return i(t,e),t}(Error);t.ConfigurationError=o;var r=function(){function e(e,t){var n=this;if(this.handle=function(e){console.log("message received from ".concat(e.origin," with data: ").concat(e.data.token)),new URL(e.origin).host==n.widgetLink.host?n.updateState(e.data.token):console.error("expected message from ".concat(n.widgetLink.host," but received message from ").concat(e.origin,". Aborting."))},this.updateState=t,e.widgetLink&&e.siteKey)throw new o;if(e.widgetLink)this.widgetLink=e.widgetLink;else{if(!e.siteKey)throw new o;e.siteKey.instanceUrl?(this.widgetLink=e.siteKey.instanceUrl,this.widgetLink.pathname="/widget/",this.widgetLink.search="?sitekey=".concat(e.siteKey.key)):this.widgetLink=new URL("https://demo.mcaptcha.org/widget/?sitekey=".concat(e.siteKey.key))}}return e.prototype.listen=function(){window.addEventListener("message",this.handle)},e.prototype.destroy=function(){window.removeEventListener("message",this.handle)},e}();t.default=r}}[607](0,e),e})(),e.exports=t()},607:function(e,t,n){"use strict";var i=this&&this.__importDefault||function(e){return e&&e.__esModule?e:{default:e}};Object.defineProperty(t,"__esModule",{value:!0}),t.ConfigurationError=t.ID=t.INPUT_NAME=void 0;var o=n(44);Object.defineProperty(t,"ConfigurationError",{enumerable:!0,get:function(){return o.ConfigurationError}});var r=i(n(44));t.INPUT_NAME="mcaptcha__token",t.ID="mcaptcha__widget-container";t.default=function(e){var n=this;this.setToken=function(e){return n.inputElement.value=e},this.receiver=new r.default(e,this.setToken),this.receiver.listen();var i=document.getElementById(t.ID);if(null==i)throw new Error("Element ".concat(t.ID,"'s parent element is undefined"));this.inputElement=document.createElement("input"),this.inputElement.id=t.INPUT_NAME,this.inputElement.name=t.INPUT_NAME,this.inputElement.hidden=!0,this.inputElement.required=!0,i.appendChild(this.inputElement);var o="mcaptcha-widget__iframe",a=document.createElement("iframe");a.title="mCaptcha",a.src=this.receiver.widgetLink.toString(),a.ariaRoleDescription="presentation",a.name=o,a.id=o,a.scrolling="no";try{a.sandbox="allow-same-origin allow-scripts allow-popups"}catch(e){try{a.sandbox.add("allow-same-origin"),a.sandbox.add("allow-scripts"),a.sandbox.add("allow-popups")}catch(e){a.setAttribute("sandbox","allow-same-origin allow-scripts allow-popups")}}a.width="100%",a.height="100%",a.frameBorder="0",i.appendChild(a)}}},t={},function n(i){var o=t[i];if(void 0!==o)return o.exports;var r=t[i]={exports:{}};return e[i].call(r.exports,r,r.exports,n),r.exports}(607);var e,t}));
```

Comment:
Do we have to ship this blob ourselves or is there an NPM package we could use?

Reply:
Since we aren't using SSR, I'm pretty sure you could run the code from the `@mcaptcha/vanilla-glue` NPM package inside the setup tag, using the `onMounted()` lifecycle hook